spec/services/support_interface/sites_export_spec.rb

```diff
@@ -42,12 +42,16 @@
           site_code: provider1.sites.first.code,
           provider_code: provider1.code,
           distance_from_provider: '64.9',
+          site_uuid: provider1.sites.first.uuid,
```

Comment:
We can DRY up `provider1.sites.first` as it gets a bit of a workout here. Likewise `provider2.sites.first` below.